app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class UsersController < ApplicationController
+  def index
+    User.all
```

Comment:
- [ ] Great progress on your project! However, there's a small issue that needs attention. In the `index` view file, you're using `@set_users` to display all users. However, in the `index` method of the `UsersController`, you're not utilizing the `set_users` method.

To address this, make sure to call `set_users` in the `index` method, like so:

```ruby
class UsersController < ApplicationController
  before_action :set_user, only: [:show, :edit, :update, :destroy]

  def index
    set_users
  end

  # ... rest of the controller code
end
```

This ensures that `set_users` is called when rendering the `index` view, and you'll see the expected list of users. Keep up the good work! 🚀👍